src/main/java/com/programmerdan/minecraft/banstick/commands/UntangleCommand.java

```diff
@@ -88,27 +88,24 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
     }
 
     public static UUID resolveName(String input) {
-        UUID playerId = null;
         if (input.length() <= 16) {
             // interpret as player name
             try {
                 return NameAPI.getUUID(input);
             } catch (NoClassDefFoundError ncde) {
             }
-            if (playerId == null) {
-                Player match = Bukkit.getPlayer(input);
-                if (match != null) {
-                    return match.getUniqueId();
-                } else {
-                    OfflinePlayer offPlay = Bukkit.getOfflinePlayer(input);
-                    if (offPlay != null) {
-                        return offPlay.getUniqueId();
-                    }
+              Player match = Bukkit.getPlayer(input);
+              if (match != null) {
+                  return match.getUniqueId();
+              } else {
+                  OfflinePlayer offPlay = Bukkit.getOfflinePlayer(input);
+                  if (offPlay != null) {
+                      return offPlay.getUniqueId();
+                   }
                 }
-            }
         } else if (input.length() == 36) {
             try {
-                playerId = UUID.fromString(input);
```

Comment:
Embarrassed I didn't catch this on original contribution. Glad you caught it here.